src/http-middlewares/before/prepare-request.js

```diff
@@ -101,9 +101,8 @@ const prepareRequest = function(req) {
 
   req = _.defaults(req, {
     merge: true,
-    // TODO: client will eventually send `removeMissingValuesFrom` rather than `omitEmptyParams`
     removeMissingValuesFrom: {
-      params: !!req.omitEmptyParams,
+      params: false,
```

Comment:
Maybe we should use `_.defaultsDeep` because we have a nested object?

Reply:
Had [considered that](https://github.com/zapier/zapier-platform-core/pull/137#discussion_r258030385). I thought we agreed on this approach, but can easily change if you think it's better with `_.defaultsDeep`.

Reply:
Ah, totally missed it. So maybe we can drop a comment here saying something like "if one of these defaults to true, we'll need to use _.defaultsDeep".